app/livechat/server/api/v1/visitor.js

```diff
@@ -128,6 +128,29 @@ API.v1.addRoute('livechat/visitor/:token/room', { authRequired: true }, {
 	},
 });
 
+API.v1.addRoute('livechat/visitor.callStatus', {
+	post() {
+		try {
+			check(this.bodyParams, {
+				token: String,
+				callStatus: String,
+				rid: String,
+			});
+
+			const { token, callStatus, rid } = this.bodyParams;
+			const guest = findGuest(token);
+			if (!guest) {
+				throw new Meteor.Error('invalid-token');
+			}
+			const status = callStatus;
+			Livechat.updateCallStatus('', rid, status, '');
```

Comment:
```suggestion
			Livechat.updateCallStatus('', rid, status);
```

The forth param is a user object, so a empty string won't be compatible